recipes/hmftools-chord/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "2.1.0_beta" %}
+{% set sha256_jar = "26d0a40e20635c4b797d28e0c7819b1653c9f8fa28180b9a556215d22387968c" %}
+{% set sha256_src = "ed039e7b405130a1e52df71c3535c6a5ed8aab0f80724bc83df88ae14aeaa257" %}
+
+package:
+  name: hmftools-chord
+  version: '{{ version }}'
+
+source:
+  - folder: jar
+    url: https://github.com/hartwigmedical/hmftools/releases/download/chord-v{{ version }}/chord-{{ version }}.jar
+    sha256: '{{ sha256_jar }}'
+  - folder: src
+    url: https://github.com/hartwigmedical/hmftools/archive/refs/tags/chord-v{{ version }}.tar.gz
+    sha256: '{{ sha256_src }}'
+
+build:
+  noarch: generic
+  number: 0
+  run_exports:
+    - {{ pin_subpackage("hmftools-chord", max_pin="x.x") }}
+
+requirements:
+  host:
+    - r-base
+    - r-randomforest
+    - r-stringr
+    - bioconductor-bsgenome
+    - bioconductor-bsgenome.hsapiens.ucsc.hg19
+    - bioconductor-bsgenome.hsapiens.ucsc.hg38
+  run:
+    - openjdk >=8
+    - r-base
+    - r-randomforest
+    - r-stringr
+    - bioconductor-bsgenome
+    - bioconductor-bsgenome.hsapiens.ucsc.hg19
+    - bioconductor-bsgenome.hsapiens.ucsc.hg38
+
+test:
+  commands:
+    - $R -e "library('CHORD')"
+    - $R -e "library('mutSigExtractor')"
+    - 'chord com.hartwig.hmftools.chord.ChordRunner -version | grep CHORD'
+
+about:
+  home: https://github.com/hartwigmedical/hmftools/blob/master/chord/
+  license: GPL-3.0-only
```

Comment:
No 'license_file' ?

Reply:
I've added a LICENSE along with the `license_file` entry in meta.yaml - let me know if this doesn't look right!